website/docs/tutorials/deploying-java-app-with-database.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Deploying a Java OpenLiberty application with PostgreSQL
+sidebar_position: 1
+---
+
+This scenario illustrates deploying a Java application with odo and linking it to an in-cluster PostgreSQL service in the minikube environment.
+
+There are two roles in this example:
+
+1. Cluster Admin - Prepare the cluster by installing the required operators into the cluster.
+2. Application Developer - Imports a Java application, creates a Database instance, and connect the application to the Database instance.
+
+## Cluster admin
+---
+
+This section assumes that you have installed minikube and configured it. See the section on setting up a [Kubernetes](../getting-started/cluster-setup/kubernetes.md) cluster using minikube if you have not already configured it.
+
+The cluster admin must install two Operators into the cluster:
+
+1. PostgreSQL Operator
+2. Service Binding Operator
+
+An operator helps in deploying instances of a given service, for example PostgreSQL, MySQL, Redis.
+
+Furthermore, these operators are "bind-able" as they expose information necessary for an application to connect to their instances.
+
+See the [Operator installation guide](../getting-started/cluster-setup/kubernetes.md) to install and configure an operator in the Kubernetes cluster, if you have not already done so.
+
+We will use [Dev4Devs PostgreSQL Operator](https://operatorhub.io/operator/postgresql-operator-dev4devs-com) found on the [OperatorHub](https://operatorhub.io) to demonstrate a sample use case.
+
+## Application Developer
+---
+
+This section assumes that you have installed `odo`. See the [Installation guide](../getting-started/installation.md) if you have not already installed it.
+
+Since the PostgreSQL Operator installed in above step is available only in `my-postgresql-operator-dev4devs-com` namespace, ensure that `odo` uses this namespace to perform any tasks:
+
+```shell
+odo project set my-postgresql-operator-dev4devs-com
+```
+### Importing the demo Java MicroService JPA application
+
+In this example we will use odo to manage a sample [Java MicroServices JPA application](https://github.com/OpenLiberty/application-stack-samples.git).
+
+1. Clone the sample application to your system:
+    ```shell
+    git clone https://github.com/OpenLiberty/application-stack-samples.git
+    ```
+
+2. Go to the sample JPA app directory:
+    ```shell
+    cd ./application-stack-samples/jpa
+    ```
+
+3. Initialize the project:
+    ```shell
+    odo create java-openliberty mysboproj
+    ```
+   `java-openliberty` is the type of your application and `mysboproject` is the name of your application.
+
+4. Push the application to the cluster:
+    ```shell
+    odo push --show-logs
+    ```
+
+5. The application is now deployed to the cluster - you can view the status of the cluster, and the application test results by streaming the cluster logs into the terminal.
```

Comment:
`odo log` doesn't show the status of the cluster. And streaming the logs doesn't happen unless one uses `--follow` flag with `odo log`. And it doesn't stream "cluster logs". It streams the logs of the component that we pushed to the cluster in the previous step.